lib/oppen.rb

```diff
@@ -38,10 +38,18 @@ def self.print(config: Config.oppen, space: ' ',
 
   # Config.
   class Config
-    # IndentAnchor.
+    # IndentAnchor (cf. test/indent_anchor_test.rb).
+    # The different ways of handling the indentation of nested groups.
     #
-    # ON_BREAK => anchor on break position (as in Oppen's original paper)
-    # ON_BEGIN => anchor on begin block position
+    # ON_BREAK => Anchor on break position (cf. Oppen's original paper).
+    #             In the case of a new line in a nested group,
+    #             the next string token will be displayed with
+    #             indentation = previous line width + last group indentation.
+    #
+    # ON_BEGIN => Anchor on begin block position.
+    #             In the case of a new line in a nested group,
+    #             the next string token will be displayed with
+    #             indentation = the sum of the indentations of all its parent groups.
```

Comment:
I think we should change the names of `ON_BREAK` and `ON_BEGIN`  since they are not explicit enough for a user that is not familiar with Oppen's algorithm. I propose `ON_BREAK => ON_LAST_LINE` and `ON_BEGIN => ON_LEFT_MARGIN`.

Reply:
These names were not present in the original paper, we choose them for our configs.

We decided to apply the following changes to the names:

`ON_BREAK => END_OF_PREVIOUS_LINE` 
`ON_BEGIN => CURRENT_OFFSET`